about-style.css

```diff
@@ -0,0 +1,503 @@
+  text-align: justify;
+}
+
+.date p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 40px;
+  color: #050000;
+  text-align: center;
+}
+
+.date h4 {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 25px;
+  text-decoration: underline;
+  text-align: center;
+}
+
+/* main-section */
+main {
+  border-top: 2px solid #f0f0f0;
+  border-bottom: 2px solid #f0f0f0;
+}
+
+.m-items {
+  width: 80%;
+  margin: 0 auto;
+  padding: 30px 0;
+}
+
+.heads h3 {
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+.heads hr {
+  border: 1px solid var(--secondary);
+  width: 20px;
+  margin: 15px auto;
+}
+
+.cards {
+  padding: 40px 12px;
+  border: 2px solid #f0f0f0;
+  margin: 20px auto;
+  max-width: 400px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* post section */
+.posts {
+  width: 80%;
+  margin: 0 auto;
+  padding: 30px 0;
+}
+
+.post h3 {
+  font-weight: 600;
+  font-size: 1.5em;
+  line-height: 35px;
+  text-align: center;
+}
+
+.post p {
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 25px;
+  text-align: center;
+  color: black;
+}
+
+.post hr {
+  border: 1px solid var(--secondary);
+  width: 20px;
+  margin: 15px auto;
+}
+
+.p-card {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin: 30px auto;
+  max-width: 750px;
+}
+
+.pcard-item {
+  margin: 0 auto;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.post-bg {
+  background: url('./images/about-post.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: white;
+  width: 100%;
+  height: 180px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+}
+
+.post-bg-2 {
+  background: url('./images/about-post-2.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: white;
+  width: 100%;
+  height: 180px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+}
+
+.overlay {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  background-color: var(--secondary);
+  opacity: 0.7;
+  z-index: 10;
+}
+
+.pcard-content {
+  margin: 40px 20px;
+  display: flex;
+  gap: 16px;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.past {
+  width: 350px;
+}
+
+/* sponsors section */
+.partners {
+  padding: 30px 0;
+  background-color: #414246;
+  display: block;
+  height: auto;
+}
+
+.part-body {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.part-head {
+  margin-bottom: 40px;
+}
+
+.part-head h3 {
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 28px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--graish);
+}
+
+.part-head hr {
+  border: 1px solid var(--secondary);
+  width: 40px;
+  margin: 15px auto;
+}
+
+.brands {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 40px;
+}
+
+footer {
```

Comment:
- [ ] Kindly fix the background color of your footer on the about page
- [ ] Desktop version
<img width="1440" alt="Screenshot 2023-03-16 at 10 25 47" src="https://user-images.githubusercontent.com/81444338/225704373-183b1541-4ee8-4479-b583-1a8e90147de0.png">

- [ ] Mobile version
<img width="502" alt="Screenshot 2023-03-16 at 10 31 23" src="https://user-images.githubusercontent.com/81444338/225704600-a0103426-c7bd-42e8-82a3-1aba46e6ab6a.png">